d4s/src/main/scala/d4s/models/DynamoException.scala

```diff
@@ -14,10 +23,10 @@ object DynamoException {
     def apply(cause: Throwable): QueryException                    = from(None, cause)
     def apply(queryName: String, cause: Throwable): QueryException = from(Some(queryName), cause)
     private def from(queryName: Option[String], cause: Throwable) = {
-      val mbName = queryName.fold(" ")(n => s" `$n` ")
+      val mbName = queryName.fold("")(n => s"`$n`")
       cause match {
-        case DynamoException(message, cause) => new QueryException(s"Dynamo query" ++ mbName ++ s"failed due to error: $message", cause)
```

Comment:
@neko-kai we will have double space if `mbName` would be empty. I'm saying this just in case, don't think that this is actually necessary.

Reply:
@aliubymov-playq Ah, I didn't get the reason this code didn't use interpolation before. Fixed 👍